javalib/src/main/scala/java/lang/IEEE754Helpers.scala

```diff
@@ -0,0 +1,68 @@
+package java.lang
+
+import scalanative.unsafe._
+import scalanative.libc.errno
+
+import scala.annotation.{switch, tailrec}
+import scalanative.posix.errno.ERANGE
+
+private[java] object IEEE754Helpers {
+
+  // Java parseDouble() and parseFloat() allow characters at & after
+  // after where C strtod() or strtof() stops. Continuous trailing whitespace
+  // with or without an initial 'D', 'd', 'F', 'f' size indicator
+  // is allowed.
+  // The whitespace can include tabs ('\t') and even newlines! ('\n').
+  // Perhaps even Unicode....
```

Comment:
"Perhaps" is not very precise.

The JavaDoc has a very clear definition of what's allowed at https://docs.oracle.com/javase/8/docs/api/java/lang/Double.html#valueOf-java.lang.String-
There is even a regex for valid inputs, which we also use in Scala.js:
https://github.com/scala-js/scala-js/blob/0e8fe18808a10887db073cf33c6211721a064e5b/javalanglib/src/main/scala/java/lang/Double.scala#L75-L85
so whitespace chars are the ones in the range `\u0000` to `\u0020` included, and only those.

Reply:
I removed the "Perhaps" line.  I have learned more about Unicode and how Java & Scala define & parse whitespace over the year since this PR was created.   

I followed the valueOf URL you kindly provided and added a reference to 
javase/8/docs/api/java/lang/String.html#trim-- to the comment.

The PR is was explicitly crafted to use the Java String.trim method to ensure consistency. 

Thank you for picking up on the less than stellar comment.  Someone may have to follow this code someday.